index.js

```diff
@@ -1,187 +1,192 @@
-                            isClosed: false,
-                        });
-                        if (!rows.length) {
-                            resFuncs.sendResponse(res, 'There isn\'t a poll open right now.');
-                        } else {
-                            const poll = pollFuncs.getPollfromResultRows(rows);
-                            const displayText = pollFuncs.getFormattedPollResults(poll);
-                            res.status(200).send({
-                                text: displayText,
-                            });
-                        }
-                        break;
-                    case 'close':
-                        resFuncs.sendResponse(res, 'We are working on closing your poll.');
-                        const pollId = await polls.closePoll({
-                            teamId: requestBody.team_id,
-                            teamName: requestBody.team_domain,
-                            channelId: requestBody.channel_id,
-                            channelName: requestBody.channel_name
-                        });
-                        const response = await polls.getByIdWithResults(pollId);
-                        if (response.length > 0) {
-                            const poll = pollFuncs.getPollfromResultRows(response);
-                            if (poll.isGame) {
-                                gameFuncs.closeNewGamePoll(res, requestBody.response_url, poll);
-                            } else {
-                                const pollResults = `Poll closed! \n${pollFuncs.getFormattedPollResults(poll)}`;
-                                resFuncs.sendDelayedPublicResponse(requestBody.response_url, pollResults);
-                            }
-                        } else {
-                            res.status(500).send(response);
-                        }
-                        break;
-                    case 'vote':
-                        pollFuncs.vote(res, requestBody, commandArray);
-                        break;
-                    case "unvote":
-                    case "remove":
-                    case "annul":
-                    case "rescind":
-                    case "repeal":
-                        pollFuncs.unvote(res, requestBody);
-                        break;
-                    default:
-                        resFuncs.sendErrorResponse(res);
-                        break;
-                }
+  const slashCommand = req.body.command
+  switch (slashCommand) {
+    case '/ww':
+    case '/werewolf':
+      const requestBody = req.body
+      const commandArray = requestBody.text.split(' ')
+      if (commandArray.length) {
+        switch (commandArray[0]) {
+          case 'poll':
+            pollFuncs.createNewPoll(res, requestBody, commandArray)
+            break
+          case 'game':
+            gameFuncs.startNewGame(res, requestBody)
+            break
+          case 'results':
+          case 'r':
```

Comment:
can you swap the order of case 'r' and case 'results'so that its slightly more verbose as to what this functionality does?